apps/provider-scaffold/template/provider/provider.go.template

```diff
@@ -0,0 +1,178 @@
+package provider
+
+import (
+	"errors"
+	"strconv"
+
+	"go.mondoo.com/cnquery/llx"
+	"go.mondoo.com/cnquery/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/providers-sdk/v1/plugin"
+	"go.mondoo.com/cnquery/providers-sdk/v1/upstream"
+	"{{ .GoPackage }}/connection"
+	"{{ .GoPackage }}/resources"
+)
+
+type Service struct {
+	runtimes         map[uint32]*plugin.Runtime
+	lastConnectionID uint32
+}
+
+func Init() *Service {
+	return &Service{
+		runtimes:         map[uint32]*plugin.Runtime{},
+		lastConnectionID: 0,
+	}
+}
+
+func (s *Service) ParseCLI(req *plugin.ParseCLIReq) (*plugin.ParseCLIRes, error) {
+	flags := req.Flags
+	if flags == nil {
+		flags = map[string]*llx.Primitive{}
+	}
+
+	conn := &inventory.Config{
+		Type: req.Connector,
```

Comment:
Only a suggestion, as we use this in some of our providers.

Add:
```
Options: map[string]string{},
```

Reply:
That is a great point, lets initialize it